src/services/entity_repository.js

```diff
@@ -145,8 +154,21 @@ export default class EntityRepository {
     const events = await this.db.collection('events').find(thisBundleStubQuery)
       .toArray();
     const entities = this.orderedEntityIds(assets, events);
-    await this.setEntitiesBundles(entities.slice(0, bundleSizeLimit), bundleId);
-    await this.unsetEntitiesBundlesStubs(entities.slice(bundleSizeLimit), bundleId);
+    await this.setEntitiesBundles(entities, bundleId);
+  }
+
+  async revokeEntitiesFromBundling(bundleStubId) {
+    const thisBundleStubQuery = {
+      'repository.bundleStubId': bundleStubId
+    };
+
+    const assets = await this.db.collection('assets').find(thisBundleStubQuery)
+      .toArray();
+    const events = await this.db.collection('events').find(thisBundleStubQuery)
+      .toArray();
+    const entities = this.orderedEntityIds(assets, events);
```

Comment:
There is no need to sort entities. Maybe we can split orderedEntityIds method to map and sort?